test/e2e/framework/pod/utils.go

```diff
@@ -130,3 +137,82 @@ func GetRestrictedContainerSecurityContext() *v1.SecurityContext {
 		Capabilities:             &v1.Capabilities{Drop: []v1.Capability{"ALL"}},
 	}
 }
+
+var psaEvaluator, _ = psapolicy.NewEvaluator(psapolicy.DefaultChecks())
+
+// MustMixinRestrictedPodSecurity makes the given pod compliant with the restricted pod security level.
+// If doing so would overrwrite existing non-conformant configuration, a test failure is triggered.
+func MustMixinRestrictedPodSecruity(pod *v1.Pod) *v1.Pod {
+	err := MixinRestrictedPodSecruity(pod)
+	gomega.ExpectWithOffset(1, err).NotTo(gomega.HaveOccurred())
```

Comment:
if this trips, does it give context of where it was called from so the caller can figure out their issue?